app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    to_return = []
+    for describe in people:
+        current_person = Person(describe["name"], describe["age"])
+
+        if "wife" in describe and describe["wife"] in Person.people:
+            create_marriage_by_wife(current_person, describe["wife"])
+
+        if "husband" in describe and describe["husband"] in Person.people:
+            create_marriage_by_husband(current_person, describe["husband"])
```

Comment:
Not correct because describe["husband/wife"] will equal to str. But you must set not name, you must set instance of another Person

Reply:
Okay, explain me please why we should write code in this way?
"it is a dict that stores Person instances by their name"

Reply:
@ArsenPidhoretskyi